### PR TITLE
Updating the Hybrid AAD join documentation

### PR DESCRIPTION
**Pull request by ghost**
Through this pull request the following changes are being made 

- Updating the SMB documentation on how to enroll devices to domain joined devices to Microsoft Intune using the new simplified method 
- Updating the git ignore file to exclude the .vs directory

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/business/manage-windows-devices.md [hunk 1]
```diff
@@ -72,44 +72,32 @@ On the Microsoft Intune page, select **Device enrollment** and on the **Overview
...
-4.	On the **Connect to Azure AD** page, enter the credentials of a global administrator for Microsoft 365 Business Premium.
-5.	On the **Device options** page, select **Configure Hybrid Azure AD join**, and then select **Next**.
-6.	On the **SCP** page, for each forest where you want Azure AD Connect to configure the SCP, complete the following steps, and then select **Next**:
-    - Check the box beside the forest name. The forest should be your AD domain name.
-    - Under the **Authentication Service** column, open the dropdown and select matching domain name (there should only be one only option).
-    - Select **Add** to enter the domain administrator credentials.  
-7.	On the **Device operating systems** page, select Windows 10 or later domain-joined devices only.
-8.	On the **Ready to configure** page, select **Configure**.
-9.	On the **Configuration complete** page, select **Exit**.
+You can install this module by invoking the following from an instance of PowerShell
```
Comment (JohanFreelancer9) [inline microsoft-365/business/manage-windows-devices.md]:
```suggestion
You can install this module by invoking the following from an instance of PowerShell:
```

File: microsoft-365/business/manage-windows-devices.md [hunk 2]
```diff
@@ -72,44 +72,32 @@ On the Microsoft Intune page, select **Device enrollment** and on the **Overview
...
+```powershell
+Install-Module SecMgmt
+```
 
-## 5. Create a GPO for Intune Enrollment – ADMX method
+> [!IMPORTANT]
+> It is recommended that you install this module on the Windows Server running Azure AD Connect.
 
-Use .ADMX template file.
+To create the required service connection point and group policy, you will invoke the  [Initialize-SecMgmtHybirdDeviceEnrollment](https://github.com/microsoft/secmgmt-open-powershell/blob/master/docs/help/Initialize-SecMgmtHybirdDeviceEnrollment.md) cmdlet. You will need your Microsoft 365 Business Premium global admin credentials when performing this task. When you are ready to create the resources invoke the following
```
Comment (JohanFreelancer9) [inline microsoft-365/business/manage-windows-devices.md]:
```suggestion
To create the required service connection point and group policy, you will invoke the  [Initialize-SecMgmtHybirdDeviceEnrollment](https://github.com/microsoft/secmgmt-open-powershell/blob/master/docs/help/Initialize-SecMgmtHybirdDeviceEnrollment.md) cmdlet. You will need your Microsoft 365 Business Premium global admin credentials when performing this task. When you are ready to create the resources, invoke the following:
```

File: microsoft-365/business/manage-windows-devices.md [hunk 3]
```diff
@@ -72,44 +72,32 @@ On the Microsoft Intune page, select **Device enrollment** and on the **Overview
...
+The first command will establish a connection with the Microsoft cloud, and when you are prompted specify your Microsoft 365 Business Premium global admin credentials.
 
-## 6. Deploy the Group Policy
+## 5. Link the Group Policy
 
-1.	In the Server Manager, under **Domains** > Group Policy objects, select the GPO from Step 3 above, for example “Cloud_Enrollment”.
-2.	Select the **Scope** tab for your GPO.
-3.	In the GPO’s Scope tab, right-click the link to the domain under **Links**.
-4.	Select **Enforced** to deploy the GPO and then **OK** in the confirmation screen.
+1. In the Group Policy Management Console (GPMC) right click on the location where you want to link the policy and select *Link an existing GPO...* from the context menu
```
Comment (JohanFreelancer9) [inline microsoft-365/business/manage-windows-devices.md]:
```suggestion
1. In the Group Policy Management Console (GPMC), right-click on the location where you want to link the policy and select *Link an existing GPO...* from the context menu.
```

File: microsoft-365/business/manage-windows-devices.md [hunk 4]
```diff
@@ -72,44 +72,32 @@ On the Microsoft Intune page, select **Device enrollment** and on the **Overview
...
+2. Select the policy created in the above step then click **OK**
```
Comment (JohanFreelancer9) [inline microsoft-365/business/manage-windows-devices.md]:
```suggestion
2. Select the policy created in the above step, then click **OK**.
```

File: microsoft-365/business/manage-windows-devices.md [hunk 5]
```diff
@@ -72,44 +72,32 @@ On the Microsoft Intune page, select **Device enrollment** and on the **Overview
...
-5.	In the **Group Policy Management Editor**, go to **Computer Configuration** > **Policies** > **Administrative Templates** > **Windows Components** > **MDM**.
-6. Right-click **Enable automatic MDM enrollment using default Azure AD credentials** and then select **Enabled** > **OK**. Close the editor window.
+```powershell
+PS C:\> Connect-SecMgmtAccount
+PS C:\> Initialize-SecMgmtHybirdDeviceEnrollment -GroupPolicyDisplayName 'Device Management'
+```
 
-> [!IMPORTANT]
-> If you do not see the policy **Enable automatic MDM enrollment using default Azure AD credentials**, see [Get the latest Administrative Templates](#get-the-latest-administrative-templates).
+The first command will establish a connection with the Microsoft cloud, and when you are prompted specify your Microsoft 365 Business Premium global admin credentials.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/manage-windows-devices.md]:
```suggestion
The first command will establish a connection with the Microsoft cloud, and when you are prompted, specify your Microsoft 365 Business Premium global admin credentials.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks

**Review by Sirkkuw — COMMENTED** [review]
I am not sure what is going on here and why you made the changes to PowerShell - I just recently updated these docs to what they are at PGs request, Further - I am not sure about some of the support files you changed.

**Pull request merged**